sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlanInfo.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.execution
 
 import org.apache.spark.annotation.DeveloperApi
+import org.apache.spark.sql.execution.command.{DataWritingCommand, DataWritingCommandExec}
```

Comment:
Do we need `DataWritingCommand` here, too?

Reply:
Will remove it